i18n/en.pot

```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2020-03-09T19:35:56.143Z\n"
-"PO-Revision-Date: 2020-03-09T19:35:56.143Z\n"
+"POT-Creation-Date: 2020-05-08T10:24:17.432Z\n"
```

Comment:
This is an autogenerated file for i18n, created when you run/build the app. I tried to gitignore it at some point but ran into trouble so let's just live with it for the time being.

Reply:
Arent those generated and merged to master automatically? In case this is true. 

I would discard the changes that has been introduced here and I would expect to `master` to  have those changes already when we merge back to it.
Am I missing something?

Reply:
The pot file is not merged to master automatically, I believe your thinking about the po files.

Reply:
yes you right, I dont really know how these things work tbh. maybe I need to ask someone to give me an intro at some point.